### DISCUSSION TITLE
What is long form field supported MySQL data type

### DISCUSSION THREAD

**Discussion by samg666** [Budibase/budibase#6460] [General]
Hello,

I have a form binded schema of MySQL table, and one of the field `Remark` is `TEXT` data type in MySQL, but when I try to bind to a `Long Form Field` it always got error `This Field setting is the wrong data type for this component`. I tried change MySQL field to `LONGTEXT` and `VARCHAR` and re-fretch the table, but still the same. May I know what kind of data type does the `Long Form Field` supported for MySQL?

<img width="402" alt="image" src="https://user-images.githubusercontent.com/29649651/175780770-c771ddb0-e7e8-49cc-9208-2ce31a4b7727.png">

**Comment by melohagan** [comment]
Hi @ggs331 

When you fetch tables, the *Text* Budibase type is chosen by default for all text related MySQL data types.

You need to manually set the column type as *Long Form Text*:
![Screenshot 2022-06-27 at 14 53 27](https://user-images.githubusercontent.com/101575380/175958201-e2dda175-1606-4942-844e-e11a9940d00a.png)

**Comment by samg666** [comment]
Thank you @melohagan , I just found that I can edit the column type 🤣 

<img width="264" alt="image" src="https://user-images.githubusercontent.com/29649651/175988466-7fc19205-f290-4706-9151-8845543590ba.png">
